### PR TITLE
LRQA-75686 | master

### PR DESCRIPTION
**Pull request by Kvale1733**
https://issues.liferay.com/browse/LRQA-75686

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:sf

**Comment by Kvale1733** [comment]
ci:stop

**Comment by Kvale1733** [comment]
ci:test:sf

File: build-test.xml [hunk 1]
```diff
@@ -15043,6 +15043,37 @@ url=${short.hostname}:8080</echo>
...
+		<if>
+			<not>
+				<isset property="database.version" />
+			</not>
+			<then>
+				<var name="database.version" value="14.4" />
+			</then>
+		</if>
+
+		<delete file="test.${user.name}.properties" />
```
Comment (vicnate5) [inline build-test.xml:15058]: We should not be deleting custom files for testers, would be safer to append their file.

**Comment by vicnate5** [comment]
@Kvale1733 I like that you are trying to use existing logic as much as possible, that is a good direction. Instead of modifying properties files, it would be better to just set the properties inside the target, then you can pass them in where they are needed. 

Here is a non working example, but might point you in the right direction
```
	<target name="start-docker-database-postgresql">
		<if>
			<not>
				<isset property="database.version" />
			</not>
			<then>
				<var name="database.version" value="14.4" />
			</then>
		</if>

		<stop-docker-database />

		<antcall inheritAll="false" target="start-docker-database">
			<param name="database.type" value="postgresql" />
			<param name="database.host" value="0.0.0.0" />
			<param name="database.version" value="${database.version}" />
			<param name="database.postgresql.docker.image" value="postgres:${database.version}" />
			<param name="database.postgresql.docker.run.options" value="--env=POSTGRES_USER=root --expose=5432 -p 5432:5432 --env=POSTGRES_HOST_AUTH_METHOD=trust" />
			<param name="database.postgresql.name" value="PostgreSQL" />
		</antcall>

		<antcall target="rebuild-database">
			<param name="database.type" value="postgresql" />
			<param name="sql.dir" value="sql/create-bare/" />
			<param name="sql.file" value="create-bare-postgresql.sql" />
		</antcall>

		<antcall target="prepare-portal-ext-properties">
			<param name="database.type" value="postgresql" />
			<param name="database.version" value="${database.version}" />
			<param name="database.host" value="0.0.0.0" />
		</antcall>
	</target>
```

Worst case you can globally set the properties in the target: `<property name="database.type" value="postgresql" />`

Note that Ant [Properties are immutable: whoever sets a property first freezes it for the rest of the build; they are most definitely not variables.](https://ant.apache.org/manual/Tasks/property.html)

**Comment by vicnate5** [comment]
ci:close

**Pull request closed without merging**